content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -281,6 +281,52 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
 or stand-alone.  In either version, it doesn't require any configuration or extra code
 to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
 
+## IPv4/IPv6 dual stack
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Alpha features change rapidly. {{< /warning >}}
+
+IPv4/IPv6 dual stack enabled Kubernetes clusters allow the assignment of both IPv4 and IPv6 addresses simulatenously which enables the following capabilities:
+
+   * Awareness of multiple IPv4/IPv6 address assignments per Pod
+   * Native IPv4-to-IPv4 in parallel with IPv6-to-IPv6 communications to, from, and within a cluster
+
+Details on the implementation of this feature may be found in the [IPv4/IPv6 dual stack KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20180612-ipv4-ipv6-dual-stack.md)
+
+### Supported Features
+
+Enabling IPv4/IPv6 dual stack on a Kubernetes cluster enables the following features:
+
+   * Dual stack Pod networking (multi-IP Pod)
+   * Kubenet multi address familiy support (IPv4 and IPv6)
+   * Egress Pod Internet routing via IPv4/IPv6 addresses
+
+### Prerequisites
+
+The following prerequisites are needed in order to utilize IPv4/IPv6 dual stack Kubernetes clusters:
+
+   * Provider support for dual stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
+   * Kubenet network plugin
+
+### Enable IPv4/IPv6 dual stack
+
+To enable IPv4/IPv6 dual stack, the following modifications must be made to the following cluster componenets:
+
+   * kube-controller-manager:
+      * `--feature-gates="IPv6DualStack=true"`
```

Comment:
As we move to changing `kube-proxy` we will ask users to enable the feature gate there as well. Might worth just saying `enable the feature gate on all components`

Reply:
I would rather only call out the affected components to avoid feature-gate sprawl.